test/test_projects/file-generate/.gitignore

```diff
@@ -0,0 +1,77 @@
+# Temporary Build Files
```

Comment:
Can we please name this directory `test/convention/...`?

Reply:
This is not containing convention but actual 'light' project using conventions, not sure it would be meaningful to call it `convention`.

Reply:
What was in my head was: this is used to test (a component of) the `golang-osd-operator` convention. But take your point that it's a "test project". Can we just call it `test/project/...` then? Again, I feel like `test_` is redundant since the subdir is already `test/`. (But if you do end up with a name with multiple words, please remember to use kebab-case.)